### ISSUE TITLE
has_next_page always returns False

### ISSUE THREAD

**Issue by gioxc88** [Shopify/shopify_python_api#388]
I am doing a simple test.

`orders = shopify.Order.find()`

this returns a list of 50 orders but if I call `orders.has_next_page()` this returns `False`.

The same thing happens if I do 

`orders = shopify.Order.find(limit=3)`

`orders.has_next_page()` still returns `False`.

I am trying to retrieve a list of orders between two date by using `created_at_min` and `created_at_max` but I always have the same problem: `orders.has_next_page()` returns `False` no matter what.

Any idea on how to solve this? it is unusable for me like this.
Many thanks for the help

**Comment by bjquinn** [comment]
Same here.  With limit=250, I get 80 orders, and has_next_page() is false.  That makes sense.  But with limit=25, has_next_page() is still false, which doesn't make sense.

**Comment by tanema** [comment]
Can you check the value of `next_page_url` on your queries as well because the `has_next_page` is just checking if that has a value, which it should.
Also I would check the `metadata` values on your queries. It will have a `pagination` value that has the next and previous page urls

**Comment by bjquinn** [comment]
The value of next_page_url is None

**Comment by tanema** [comment]
That indicates that you are not getting pagination headers for some reason. Like I said, in the same way you checked your `next_page_url` you should also check the value of `metadata`

**Comment by gioxc88** [comment]
The problem is with your the library.
If I try to use shopify REST API directly I get the correct result. 

Checking metadata provides the same result.
No next page

**Comment by tanema** [comment]
> The problem is with your the library.
Yes I am asking questions to figure out where that problem might lay. What is the exact contents of the metadata because more than just pagination is store in there. If it is `None` then it would not work at all

**Comment by gioxc88** [comment]
> > The problem is with your the library.
> 
> Yes I am asking questions to figure out where that problem might lay. What is the exact contents of the metadata because more than just pagination is store in there. If it is `None` then it would not work at all

Sorry about that, I didn't mean to be rude. Give some time I'll post the data you need

**Comment by DougPlumley** [comment]
@tanema I'm seeing this issue as well, it seems like the Link||link key value isn't being returned in the metadata headers so the next and previous values are not being set.

**Comment by DougPlumley** [comment]
Just realized I didn't set the API version on a new feature branch, this issue can be resolved with this snippet:

`api_version = '2020-01'
shop_url = f"https://{os.environ['SHOPIFY_URL']}.myshopify.com/admin/api/{api_version}"
shopify.ShopifyResource.set_site(shop_url)`

**Comment by pankaj-webkul** [comment]
Changing
from
`shopify.ShopifyResource.set_site(<url>/admin)` 
(shopify.ShopifyResource.connection.response.headers['X-Shopify-API-Version'] = '2019-04')
to `shopify.ShopifyResource.set_site(<url>/admin/api/2020-07)`
(shopify.ShopifyResource.connection.response.headers['X-Shopify-API-Version'] = '2020-07')
worked for me.

**Comment by rbnali** [comment]
I encountered the exact same issue with `shopify.Product.find()`.

Adding `/api/{api_version}` to the shop url as @DougPlumley suggested worked out for me as well.
